qml/keys/EscapeKey.qml

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Vojtěch Pluskal
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.4
+
+ActionKey {
+    label: "esc"
```

Comment:
If these buttons are to be included (is there a point, if nothing is using them?), they will definitely need better labels and icons. While `esc` might fit on a single key, most of the things added here do not, and should probably have icons instead.

Reply:
As I mentioned, I wanted to use a icon for the tab key, but could not find any list of supported icon names. The names of page up/down keys can be shortened to _pg up_ and _pg dn_. I also found out, that there exist symbols for _home_ and _end_ keys, but I have not seen them used yet. There would just be an issue with _caps lock_, as its symbol is already used with locked shift key. I am unsure how to solve this other than just having a wider key :).